backend/src/model/dto/new_map_impl.rs

```diff
@@ -17,6 +19,16 @@ impl From<NewMapDto> for NewMap {
             visits: new_map.visits,
             harvested: new_map.harvested,
             owner_id: new_map.owner_id,
+            privacy: new_map.privacy,
+            description: new_map.description,
+            location: match new_map.location {
+                Some(latlng) => Some(Point {
+                    x: latlng.longitude,
+                    y: latlng.latitude,
+                    srid: Some(4326),
+                }),
+                None => None,
+            },
```

Comment:
```suggestion
            location: map.location.map(|latlng| Point {
                    x: latlng.longitude,
                    y: latlng.latitude,
                    srid: Some(4326),
            }),
```

If you use `From` and `Into` traits use the suggestion described in the other file.